docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,205 @@
+---
+title: How to use go-plugin-runner with APISIX Ingress
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Brief Description
+
+Based on version 0.3 of the go-plugin-runner plugin and version 1.4.0 of APISIX Ingress, this article goes through steps as follows:
+
+1. Prepare the environment (example as follows);
+2. Create the cluster;
+3. Build a container image that includes the go-plugin-runner;
+4. Customize the helm chart package;
+5. Install and Deploy;
+6. Verify the function.
+
+It is guaranteed that the final result can be derived in full based on this environment example as follows:
+
+```bash
+go-plugin-runner: 0.3
+APISIX Ingress: 1.4.0
+kind: v0.12.0
+kubectl version(Client/Server): v1.23.5/v1.23.4
+golang: 1.18
+```
+
+## Begin
```

Comment:
I think we do not need this heading. The heading level 2 should be the 6 steps mentioned in the introduction.

Reply:
Change all headings according to this.

Reply:
Solved